code/Pagetypes/FreshdeskPage.php

```diff
@@ -0,0 +1,262 @@
+<?php
+
+
+class FreshdeskPage extends Page
+{
+}
+
+class FreshdeskPage_Controller extends Page_Controller
+{
+    /**
```

Comment:
So continuing on the API+Client merge comment, I'd be expecting to see here

```php
    private static $dependencies = [
        'freshdesk' => '%$FreshdeskAPI',
    ];
```

Reply:
ack